pyfstat/mcmc_based_searches.py

```diff
@@ -1000,11 +1000,17 @@ def plot_chainconsumer(self, save_fig=True, label_offset=0.25, dpi=300, **kwargs
 
         samples_plt = self._scale_samples(samples_plt, self.theta_keys)
 
-        import chainconsumer
+        try:
```

Comment:
Please move to start of method, before it does potentially slow stuff with samples.